projects.tex

```diff
@@ -1,36 +1,32 @@
-\section{PROJECTS}
+\section{MENTORSHIP \& PROJECTS}
 \begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
-  Fluoroscopy Simulator – Medical College of Wisconsin & September 2019 – May 2020
+  FIRST Mentor – Charger Robotics Team 537 & October 2021 – Present
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Created UI mockups detailing the user workflow and consulted with UX professors and application users to
-        iterate on the design prior to starting development work on the application.
+        Taught programming concepts and problem solving skills to students, allowing them to continue learning independently
+        and to contribute to the robot's codebase.
     }
     \item{
-        Architected the full-stack web application solution based on requirement gathering sessions with the product
-        owner and other stakeholders, improving on the original POC developed prior to our team’s involvement.
+        Worked with students and department leaders to develop leadership qualities, enabling them to teach each other and
+        become leaders whom work on developing the skills of other students to aid robot programming.
     }
     \item{
-        Researched image analysis techniques using OpenCV, documenting the process in Jupyter notebooks.
-        Viable options were produced to calculate the distance between two points in 3D space and returning the
-        image to the client as a static image.
+        Instilled FIRST Core Values and Gracious Professionalism into students, helping develop skills and behaviors to
+        promote a healthy environment for learning and fun.
     }
 \end{bulletlist}
 
 \begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
-    At The Control – FIRST Robotics Streaming Site & September 2015 – August 2017
+    Senior Design Project – MSOE Tracking & September 2020 – May 2021
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Created and altered database tables to store data obtained from the FRC Events API to minimize the update
-        requests made and allow users to see real-time event updates for live matches.
+        Develop an intelligent, scalable video analytics application to autonomously monitor, track, and react to
```

Comment:
"Develop" => "Developed"

Reply:
Good catch! Fixed in ef0a2873a79bb05b113862064512b68152d0b7b0